website/pages/en/quick-start.mdx

```diff
@@ -35,133 +30,141 @@ Using [yarn](https://yarnpkg.com/):
 yarn global add @graphprotocol/graph-cli
 ```
 
-## 3. Initialize your subgraph from existing contract
+### 2. Create your subgraph
+
+If your contract has events, the `init` command will automatically create a scaffold of a subgraph.
+
+#### Create via CLI
+
+Use the following command to create a subgraph in Subgraph Studio using CLI: `init --product subgraph-studio`
```

Comment:
We should use code blocks on this page as much as possible IMO, to let users copy the commands. Also, this is not the full command, it's missing the executable (`graph`):
````suggestion
Use the following command to create a subgraph in Subgraph Studio using Graph CLI:

```sh
graph init --product subgraph-studio
```
````